sql-plugin/pom.xml

```diff
@@ -49,41 +44,13 @@
             <artifactId>scalatest_${scala.binary.version}</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <!--Note Spark 3.0.1~3.2.2 use 1.9; 3.3.0 use 1.12; but we use 1.11-->
+        <!--TODO investigate: flat buffers is a little jar, just use version 1.9 is sufficient? better not shade it-->
```

Comment:
IIRC we needed version 11 for an API we were using.  As you said, it's small and I don't see any reason to use the Spark version here.  The reason we're using flatbuffers has nothing to do with what version Spark is using.  We're not passing flatbuffers to/from Spark, this is for communication in our UCX-based shuffle and therefore it seems appropriate to use a fixed version so we don't have to worry about flatbuffer version-specific bugs across Spark versions.